datajunction-query/djqs/engine.py

```diff
@@ -100,7 +116,11 @@ def run_query(
         cur = conn.cursor()
 
         return run_snowflake_query(query, cur)
-    sqla_engine = create_engine(engine.uri, connect_args=engine.extra_params)
+    
+    if engine.type == EngineType.SQLALCHEMY:
```

Comment:
Not sure this is correct. If the engine type is sqlalchemy, it doesn't necessarily mean it's using trino. If you want to call this specific trino engine init, I would inspect the `uri`.